content/en/docs/reference/using-api/api-overview.md

```diff
@@ -18,95 +18,115 @@ This page provides an overview of the Kubernetes API.
   - The support for a feature will not be dropped, though the details may change.
-  - The schema or semantics of objects may change in incompatible ways in a subsequent beta or stable release.
-    When this happens, migration instructions are provided. Schema changes may require deleting, editing,
-    or recreating API objects. The editing process may not be straightforward. Migration may require
-    downtime for applications that rely on the feature.
-  - The software is not recommended for production uses. Subsequent releases may introduce
-    incompatible changes. If you have multiple clusters which can be upgraded independently,
-    you may be able to relax this restriction.
-  - Try the beta features and provide feedback. After the features exit beta, it may not be practical
-    to make more changes.
 
+  - The schema and/or semantics of objects may change in incompatible ways in
+    a subsequent beta or stable release. When this happens, migration
+    instructions are provided. Schema changes may require deleting, editing, and
+    re-creating API objects. The editing process may not be straightforward.
+    The migration may require downtime for applications that rely on the feature.
+  - The software is not recommended for production uses. Subsequent releases
+    may introduce incompatible changes. If you have multiple clusters which
+    can be upgraded independently, you may be able to relax this restriction.
+  
+  {{< note >}}
+  Try the beta features and provide feedback. After the features exit beta, it
+  may not be practical to make more changes.
+  {{< /note >}}
+  
 - Stable:
   - The version name is `vX` where `X` is an integer.
   - The stable versions of features appear in released software for many subsequent versions.
 
 ## API groups
 
-[API groups](https://git.k8s.io/community/contributors/design-proposals/api-machinery/api-group.md) make it easier to extend the Kubernetes API. The API group is specified in a REST path and in the `apiVersion` field of a serialized object.
+[API groups](https://git.k8s.io/community/contributors/design-proposals/api-machinery/api-group.md)
+make it easier to extend the Kubernetes API.
+The API group is specified in a REST path and in the `apiVersion` field of a
+serialized object.
 
 Currently, there are several API groups in use:
 
-- The *core* (also called *legacy*) group is found at REST path `/api/v1`. The core group is not specified
-  as part of the `apiVersion` field, for example, `apiVersion: v1`.
-- The named groups are at REST path `/apis/$GROUP_NAME/$VERSION` and use `apiVersion: $GROUP_NAME/$VERSION`
-  (for example, `apiVersion: batch/v1`). You can find the full list of supported API groups in
-  [Kubernetes API reference](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/).
+*  The *core* (also called *legacy*) group is found at REST path `/api/v1`.
+   The core group is not specified as part of the `apiVersion` field, for
```

Comment:
When reading this section, it is not clear what "core" or "legacy" implies?
Why are there several versions of the Event API? Which version is the most recent or maintained?

Reply:
My take of this is that the so called 'API group' wasn't there in early versions of Kubernetes. The developers (especially those from the SIG API-Machinery) then felt a need to stop adding everything to the huge 'v1' group. The single API group is simply not scalable. That is the reason we had `extensions` (now disappeared) and `apps` and `batch` ... etc. Ideally, the 'v1' group can be renamed to 'core/v1', however, that will break the whole world. So it is called the 'core' group, without a group name, and it became the 'legacy' group sometimes. 

Event began its life in the 'core' group as well, now gradually migrated into its own group. Event in the new `events.k8s.io` group has just graduated from 'v1beta1' to 'v1' (GA'ed), and that is the one to use going forward. However, if you check the implementation closely, it is borrowing code from the 'core' group. Refer to: https://github.com/kubernetes/kubernetes/issues/94748